args.go

```diff
@@ -260,7 +260,7 @@ func (a *Args) PeekBytes(key []byte) []byte {
 func (a *Args) PeekMulti(key string) [][]byte {
 	var values [][]byte
 	a.VisitAll(func(k, v []byte) {
-		if string(k) == key {
+		if b2s(k) == key {
```

Comment:
This is not needed. `string(bv) == sv` is optimized in the compiler and doesn't allocate: https://go.dev/play/p/WYcbS08dv_a
I haven't tested it but I wouldn't be surprised if it's even faster than with `b2s`.

Reply:
thanks